File: app/src/main/java/mk/webfactory/template/di/AppModule.java
package mk.webfactory.template.di;

import android.app.Application;
import android.content.Context;
import dagger.Module;
import dagger.Provides;
import mk.webfactory.template.di.qualifier.ApplicationContext;

@Module
public class AppModule {

  @Provides @ApplicationContext public Context provideContext(Application application) {
    return application;
  }
}

File: app/src/main/java/mk/webfactory/template/di/AppModule.kt
package mk.webfactory.template.di

import android.app.Application
import android.content.Context
import dagger.Module
import dagger.Provides
import mk.webfactory.template.di.qualifier.ApplicationContext

@Module
class AppModule {

    @Provides
    @ApplicationContext
    fun provideContext(application: Application): Context {
        return application
    }
}